tests/bearlib/abstractions/external_bear_wrap_testfiles/test_external_bear.py

```diff
@@ -4,13 +4,13 @@
 
 # Start ignoring PycodestyleBear* as those imports may raise
```

Comment:
@sangamcse So I guess this would do:
```
# Start ignoring PycodestyleBear*, PEP8Bear as those imports may raise
import os
import sys
import json

# Start ignoring PycodestyleBear* as those imports may raise
# import warnings
from coalib.results.Result import Result
from coalib.results.SourceRange import SourceRange
from coalib.output.JSONEncoder import create_json_encoder
from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY

sys.path.append(os.path.join(os.path.dirname(__file__),
                             '..', '..', '..', '..'))
# Stop ignoring
```

We only require one `#Stop ignoring` or do I have to add one more such that:
```
# Start ignoring PycodestyleBear*, PEP8Bear as those imports may raise
import os
import sys
import json

#Stop ignoring

# Start ignoring PycodestyleBear* as those imports may raise
# import warnings
from coalib.results.Result import Result
from coalib.results.SourceRange import SourceRange
from coalib.output.JSONEncoder import create_json_encoder
from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY

sys.path.append(os.path.join(os.path.dirname(__file__),
                             '..', '..', '..', '..'))
# Stop ignoring
```

Reply:
Only this much
```py
# Start ignoring PEP8Bear*, PycodestyleBear* as those imports may raise
# import warnings
import os
import sys
import json

sys.path.append(os.path.join(os.path.dirname(__file__),
                             '..', '..', '..', '..'))

from coalib.results.Result import Result
from coalib.results.SourceRange import SourceRange
from coalib.output.JSONEncoder import create_json_encoder
from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
# Stop ignoring
```

Reply:
Btw, the build log shows that the PEP8 is not failing it is something due to `json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)` which is not showing up when ran locally.

Reply:
@bkhanale, the test is failing due to `test_path` which is definitely causing due to the changes in the imports. And that's only reason json is failing. I hope you will get it by reading all error logs. 😃